donate/payments/tasks.py

```diff
@@ -60,16 +60,16 @@ def mofo_donation_receipt_number_format(amount):
     "card_type",
     "last_4",
     "locale",
-    "service",
```

Comment:
The emailing receipt function was expecting to receive the payment method in a variable called service, however, it was sending it titled as "payment_method" instead. Have tested on local machine with Credit Card and Paypal and can confirm it is updating correctly now!